lib/Net/DRI/Protocol/EPP/Extensions/CentralNic/Fee.pm

```diff
@@ -224,16 +224,16 @@ sub fee_set_parse
     } elsif ($name eq 'fee')
     # Fees are kind of loosely defined based on free text description field with refundable also possible. This will total it up and concat the description and refundable fields but its only human readable
     {
-      $set->{fee} = 0 unless exists $set->{fee};
-      $set->{fee} += $content->textContent();
+      $set->{fee} = '0.00' unless exists $set->{fee};
+      $set->{fee} = sprintf("%.2f", ($set->{'fee'} + $content->textContent())); # wasn't appending x.00 for some reason. Using this way to avoid such problem :p
```

Comment:
Perhaps we could use a perl module to check the numbre of decimal points based on currency?